index.js

```diff
@@ -0,0 +1,43 @@
+// IMPORT MODULES
+import StorageBooks from './modules/storageBooks.js';
+import toggleVisbility from './modules/toggleVisibility.js';
+import theClasses from './modules/theClasses.js';
+import { DateTime } from './node_modules/luxon/src/luxon.js';
+
+// import { DateTime } from "luxon";
```

Comment:
- [ ] Comments make your codebase look messy. You can always use git history to get back to your previous work.